### DISCUSSION TITLE
REQUEST: add Super Game Boy

### DISCUSSION THREAD

**Discussion by ridsama** [eduardofilo/RG350_adam_image#122] [Ideas & Requests]
Super Game Boy borders work with the existing mGBA opk, it's just a matter of adding it to the Consoles and theme support.

**Comment by eduardofilo** [comment]
Do you mean with RA's mGBA core?

**Comment by ridsama** [comment]
Yes, RA mGBA core.

**Reply by eduardofilo** [reply]
OK, I take note and will consider adding SGB as a system. I anticipate that it will depend on whether I find or can make the art for that system in the themes installed in SimpleMenu.

**Comment by ridsama** [comment]
I did find the theme for SGB that matches BigCody but it's all in pieces
https://github.com/CodyV59/es-theme-epic-cody-RG351V. I don't have a 351V
(yet) to take a screenshot of the final image.

**Reply by eduardofilo** [reply]
Well, at least we already have something. Thanks for searching it.

**Comment by ridsama** [comment]
![sgb](https://user-images.githubusercontent.com/33970541/146323312-b7e93614-238d-4578-9462-ac92e3a8e692.png)
I don't know why I tried to wait for 351V to do this. I got a screenshot for the theme that I mentioned for SGB.

**Reply by ridsama** [reply]
320x240 version
![sgb_s](https://user-images.githubusercontent.com/33970541/146324997-cbebbc46-9a2c-405a-b684-e9b6523a63b0.png)

**Reply by ridsama** [reply]
in ~/.simiplemenu/section_groups/consoles.ini I added SGB in the list:

```
[CONSOLES]
consoleList = NES,FDS,SNES,SGB,...

[SGB]
execs = /media/data/apps/retroarch_rg350_GBA_mGBA.opk
romDirs = /media/sdcard/roms/SGB/
romExts = .gb,.gbc,.zip,.7z
scaling = 0
```
In ~/.simplemenu/themes/320x240/BigCody/theme.ini I add:
```
[SGB]
name = Super Game Boy
logo = resources/sgb/logo.png
```
Also add it in ~/.simplemenu/themes/640x480/BigCody/theme.ini too.

Then just place the logo above in ~/.simplemenu/themes/320x240/BigCody/resources/sgb (create sgb folder) and name the pictures logo.png (also in 640x480/BigCody/resources/sgb too).

**Reply by eduardofilo** [reply]
Perfect. If I find the art for SGB in the remain themes, I will incorporate it. Thanks...

**Reply by valzi** [reply]
Thanks for these instructions and for the image! I just did this to my rg280v and it's great!

**Comment by eduardofilo** [comment]
In which group do you think this system should go? On handhelds or consoles?

EDIT: OK, as your first comment, it should be in Consoles group.

**Comment by ridsama** [comment]
I added to consoles because you play it on Super FC/NES. But you might
think differently.

**Reply by eduardofilo** [reply]
I trust your criteria. It will be in Consoles group. Thanks.

**Reply by valzi** [reply]
I think it makes more sense as handheld because that places it right next to the GB and GBC options.
